src/operators/implementation/onnx/operator__onnx__elu__6__T_tensor_float.c

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+#include "trace.h"
+#include "utils.h"
+
+ operator_status operator__onnx__elu__6__T_tensor_float(
+     node_context *ctx
+ )
+ {
+
+   Onnx__TensorProto *X = searchInputByName(ctx, 0);
+
+   Onnx__TensorProto *Y = searchOutputByName(ctx, 0);
+
+   TRACE_LEVEL0("Calling operator_elu\n");
+
+   if (0){
+     /* TODO: Check some conditions. For example if a specific
+      * functionality is not supported */
+     return -1;
+   }
+
+   debug_print_dims(X->n_dims, X->dims);
+
+   /* Default value */
+   float alpha = 1.0;
+
+   /* If there is an attribute, its alpha */
+   if (ctx->onnx_node->n_attribute == 1){
+     alpha = ctx->onnx_node->attribute[0]->f;
+   }
```

Comment:
I don't quite get what you mean.
If you take a look at the `Onnx__AttributeProto` declaration, you see each possible type has its own field.
This is not really covered since these declaration are generated by protobuf, the source is the only thing you have.
```suggestion
   Onnx__AttributeProto *a_alpha = searchAttributeNyName(ctx->onnx_node->n_attribute,ctx->onnx_node->attribute, "alpha");
   if (a_alpha) {
     alpha = a_alpha->f;
   }
```
Yeah the documentation is thin on this one :D
I'm also not in love with this helper, but using it, allows for quicker refactoring later on.

Reply:
> I don't quite get what you mean.

See, for example, the conv implementation:
https://github.com/alrevuelta/cONNXr/blob/master/src/operators/implementation/onnx/operator__onnx__conv__11__T_tensor_float.c#L37
`kernel_shape` has an array called `ints` that holds the values.  I wasn't sure if this was the general approach to any attribute.  I see now that it is not.

I'll make a related adjustment to leakyrelu